5/README.md

```diff
@@ -82,13 +88,6 @@ The following Types SHALL be supported by Device Properties and Class Properties
 
 A conforming Tango implementation SHALL provide a way to retrieve a persisted Property Value and to convert it to the above types.
 
-Device Properties and Class Properties MAY have a _Default Value_ which can be used if the Property Value has 
-not been persisted.
-
-A Tango Device Property MAY be defined as _Mandatory in Database_. 
-This metadata can be used when the device server programmer requires this Property to be persisted.
-A Device Property defined as _Mandatory in Database_ SHALL not have a Default Value.
-
 It is RECOMMENDED to support _Not a Number (NaN)_ as well as _-infinity (-inf)_ and _+infinity (+inf)_ DevFloat and DevDouble values.
```

Comment:
I am confuse here since these values are text and not real. Although I understand the end goal it suppose that you set the property as string.
@piogor these values should also be specify in RFC-9 but I guess it will be express like NaN value such as the square root of negative numbers or the result of 0/0.

Reply:
> I am confuse here since these values are text and not real. 

No, these values are not text, at least this is not what I meant. Here what I wanted to say is that the property mechanism should allow to handle the NaN, -infinity and +infinity values. So this means that a device server programmer should be able to read a property with these special values and to interpret them correctly as NaN, -infinity and +infinity in the device server code (as defined in standards like https://en.wikipedia.org/wiki/IEEE_754).
This also means that the Tango implementation should be able to persist this kind of values.
It can be persisted as  special strings as it is now by the current implementation of the Database server but another Database server implementation could very well decide to persist it as float or double.

> Although I understand the end goal it suppose that you set the property as string.

Could you elaborate on this point? What do you mean with " you set the property as string" .
Who is "you" in this sentence?  Do you mean it should be persisted as a string?